corehq/toggles.py

```diff
@@ -2048,3 +2048,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_SOLUTIONS_LIMITED,
     namespaces=[NAMESPACE_DOMAIN]
 )
+
+
+ADD_LIMITED_FIXTURES_TO_SMS_WORKFLOW = StaticToggle(
+    'fixtures_in_sms_workflow',
+    'Allow limited fixtures to be available in SMS workflows',
```

Comment:
What are "limited fixtures"?  Also, is there a more specific feature name than "SMS workflows"?

Reply:
just said limited fixtures because we are not sharing all fixtures, can only say locations as well since that is all we are doing for now.
`ADD_LIMITED_FIXTURES_TO_CASE_RESTORE` ?

Reply:
Sure, that sounds good, thanks

Reply:
https://github.com/dimagi/commcare-hq/pull/29482/commits/68b1b42f77aa7527f0339d5d23426c4211e1cc53